### PR TITLE
feat(orb-ui): Add beacon orb-ui command

### PR DESCRIPTION
**Pull request by AlexKaravaev**

### CODE REVIEW

File: ui/src/engine/mod.rs [hunk 1]
```diff
@@ -374,6 +377,7 @@ event_enum! {
         #[event_enum(method = sound_test)]
         SoundTest,
 
+
```
Comment (TheButlah) [inline ui/src/engine/mod.rs:380]:
```suggestion
```

**Review by TheButlah — APPROVED** [review]

File: ui/src/beacon.rs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+use tracing::info;
+
+pub async fn beacon(ui: &dyn Engine, duration: Duration) -> Result<()> {
+    info!("🔹 Starting beacon");
+
+    let end_time = time::Instant::now() + duration;
+    while time::Instant::now() < end_time {
+        ui.beacon();
+        time::sleep(Duration::from_secs(1)).await;
+    }
```
Comment (fouge) [inline ui/src/beacon.rs:14]: should you reset the UI once done? 🤔

File: ui/src/engine/pearl/mod.rs [hunk 3]
```diff
@@ -223,6 +223,34 @@ impl EventHandler for Runner<PEARL_RING_LED_COUNT, PEARL_CENTER_LED_COUNT> {
             Event::Resume => {
                 self.paused = false;
             }
+            Event::Beacon => {
+                let master_volume = self.sound.volume();
+                self.sound.set_master_volume(100);
```
Comment (fouge) [inline ui/src/engine/pearl/mod.rs:228]: if i'm not mistaken, 100 is too loud on Pearl and we risk blowing the fuse

the value is clamped on the backend side on normal usage

would `50` work for you?

**Review by fouge — COMMENTED** [review]
@AlexKaravaev 
please look at my comments

**Pull request merged**